dev/com.ibm.ws.session/src/com/ibm/ws/session/SessionProperties.java

```diff
@@ -760,6 +759,14 @@ private static boolean setDefaultConfiguration(SessionManagerConfig smc, Map<Str
             smc.setSessionCookieSameSite(SameSiteCookie.get(sValue));  
         }
 
+        if (ProductInfo.getBetaEdition()) {
+            s = "cookiePartitioned";
+            bValue = propertyToBoolean(xtpProperties.get(s));
+            if (bValue != null) {
+                smc.setSessionCookiePartitioned(bValue.booleanValue());
```

Comment:
Why call booleanValue here when setSessionCookieParitioned takes a `Boolean`?

Reply:
Good point -- I'll make the fix